kolibri/plugins/management/assets/src/vue/data-page/index.vue

```diff
@@ -1,16 +1,40 @@
 <template>
 
-  <div>
+  <div class="wrapper">
+
     <h1>Export Usage Data</h1>
 
     <p>
-      Download a comma-separated spreadsheet containing information about users and
+      Download comma-separated spreadsheets containing information about users and
       their interactions with the content on this device.
     </p>
```

Comment:
I have very little idea what this sentence is about, I will prefer a more straight forward expression. The same goes to the descriptions for Content Summary Logs and Content Session Logs.